src/clj/nnts2/organization/db.clj

```diff
@@ -0,0 +1,20 @@
+(ns nnts2.organization.db
+  (:require [honeysql.helpers :as h]
+            [honeysql.core :as sql]
+            [honeysql-postgres.format :refer :all]
+            [honeysql-postgres.helpers :as ph]
+            [nnts2.config :refer [db-spec]]
+            [nnts2.middleware :refer [snake->kebab]]
+            [clojure.java.jdbc :as jdbc]))
+
+(defn insert
+  ([details] (insert details db-spec))
```

Comment:
if you require config :as config, this line can be (insert details config/db-spec), and the other places db-spec remains the same. This will make it clearer which one is the db-spec coming from config file